### PR TITLE
Always be recruiting MVP

### PR DESCRIPTION
**Pull request by marteki**
The PR updates the index page to promote the tech@cfpb.gov email address for recruiting. It also adds a privacy statement.

Preview: http://marteki.github.io/cfpb.github.io/index.html

This is a super basic MVP. @jimmynotjim and I have already started looking at ways to improve on it, such as better usage of the sidebar and using form fields to funnel directly into an e-mail message. However, looking to improve shouldn't prevent us from publishing what we have.

### CODE REVIEW

**Comment by ascott1** [comment]
:clap:

File: css/site.css [hunk 1]
```diff
@@ -631,9 +631,13 @@ hr {
     border-bottom-color: #0072CE;
 }
 .intro .learn-more {
-    margin: 0; padding: 0;
+    margin: 0;
+    padding: 0;
     list-style: none;
 }
+.intro .work-with-us {
+    padding-bottom: 0px;
```
Comment (jimmynotjim) [inline css/site.css]: No need to include units when using `0` value here.

File: index.html [hunk 2]
```diff
@@ -25,6 +25,18 @@
                                 </a>
                             </li>
                         </ul>
+                        <section class="work-with-us">
+                            <h2>Want to work with us?</h2>
+                            <p>E-mail your resume to <a href="mailto:tech@cfpb.gov?subject=Interested in working in tech at CFPB">tech@cfpb.gov</a> and
+                        we'll contact you about relevant opportunities throughout the year. Previous openings have been in the fields of cybersecurity, graphic design, web development, user experience, and data architecture/analysis.</p>
```
Comment (jimmynotjim) [inline index.html]: It's [best to add semantic line breaks](https://github.com/cfpb/front-end/blob/master/markup.md#semantic-line-breaks-in-source-code) at 80 chars here and throughout -

**Comment by jimmynotjim** [comment]
Couple of nit-picky things. If you don't have time to I can lend a hand on it tomorrow.

File: css/site.css (hunk 1 above)
Comment (marteki) [inline css/site.css]: px removed!

File: index.html (hunk 2 above)
Comment (marteki) [inline index.html]: Thanks! Added at ~80 chars throughout.

File: index.html [hunk 3]
```diff
@@ -247,5 +262,48 @@ <h3 class="sidebar_list_item_head">
...
+                        <p>
+                            E-mail your resume to 
+                            <a href="mailto:tech@cfpb.gov?subject=Interested in working in tech at CFPB">tech@cfpb.gov</a>
+                             and we'll contact you about relevant opportunities throughout the year. Previous 
+                             openings have been in the fields of cybersecurity, graphic design, web development, 
+                             user experience, and data architecture/analysis.
+                        </p>
+                        <p>
+                            Learn more about the types of work we do at our page for the
+                            <a href="http://www.consumerfinance.gov/jobs/technology-innovation-fellows/">Technology &amp; Innovation Fellowship</a><i class="icon-external-link"></i>.
```
Comment (KimberlyMunoz) [inline index.html]: Can you add a space between the `</a>` and `<i>` to give it some seperation?

**Comment by KimberlyMunoz** [comment]
On [Line 48](https://github.com/marteki/cfpb.github.io/blob/gh-pages/index.html#L48), could you change how many stories we pull in to `{% for post in site.posts limit:4 %}` so that we have less whitespace?

image

**Comment by stephanieosan** [comment]
If we do that, it will be painfully apparent how infrequently we publish, though. :sweat_smile:

**Comment by jimmynotjim** [comment]
I'd say it might be a good shove to publish more often, but in reality, the job stuff will be moving to that space, so there's no point messing with it now.

**Comment by KimberlyMunoz** [comment]
Okay, no need to change then. :smile:

File: index.html (hunk 3 above)
Comment (marteki) [inline index.html]: Done!

**Pull request merged**